src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -83,6 +83,11 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack
 		) {
+			if( type is IErrorTypeSymbol  || type == null ) {
+				// type can't be resolved; the project is either not configured correctly, or there's a typo
+				return false;

```

Comment:
At the very least, this behaviour is consistent with what happens currently (type.GetMembers() returns empty list, type gets reported as safe).